src/index.js

```diff
@@ -157,6 +157,7 @@ function EvolvClient(options) {
    * @param flush {Boolean} If true, the event will be sent immediately.
    */
   this.emit = function(type, metadata, flush) {
+    context.pushToArray('events', {type: type,  timestamp: new Date().getTime()})
```

Comment:
No need to change, just a more succinct alternative:
```suggestion
    context.pushToArray('events', { type: type,  timestamp: Date.now() })
```

Reply:
Done